src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -2888,11 +2888,15 @@ class CommentOnStatementSegment(BaseSegment):
                         "ROUTINE",
                     ),
                     Ref("ObjectReferenceSegment"),
-                    Bracketed(
-                        Sequence(
-                            # TODO: Is this too permissive?
-                            Anything(),
+                    Sequence(
+                        Bracketed(
+                            Sequence(
+                                # TODO: Is this too permissive?
+                                Anything(),
+                                optional=True,
+                            ),
                         ),
+                        optional=True,
```

Comment:
I don't think you need the extra `Sequence` as the `Bracketed segment can just be optional:

```suggestion
                    Bracketed(
                        Sequence(
                            # TODO: Is this too permissive?
                            Anything(),
                            optional=True,
                        ),
                        optional=True,
                    ),
```

Reply:
Thanks!